docs/templates/index.rst.jinja2

```diff
@@ -83,19 +83,15 @@ but override the ``database`` and ``schema`` values:
 Note that when using a profile mapping, the profiles.yml file gets generated with the profile name and target name
 you specify in ``ProfileConfig``.
 
-Disabling dbt event tracking
+Dbt config vars
 --------------------------------
-.. versionadded:: 1.3
+.. versionadded:: 1.3.2
 
-By default `dbt will track events <https://docs.getdbt.com/reference/global-configs/usage-stats>`_ by sending anonymous usage data
-when dbt commands are invoked. Users have an option to opt out of event tracking by updating their ``profiles.yml`` file.
-
-If you'd like to disable this behavior in the Cosmos generated profile, you can pass ``disable_event_tracking=True`` to the profile mapping like in
-the example below:
+The parts of ``profiles.yml``, which aren't specific to a particular data platform `dbt docs <https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml>`_
 
 .. code-block:: python
 
-    from cosmos.profiles import SnowflakeUserPasswordProfileMapping
+    from cosmos.profiles import SnowflakeUserPasswordProfileMapping, DbtConfigVars
```

Comment:
Need to keep disable event tracking docs, but you can add Dbt config vars for release 1.4.0. Would also suggest showing an example with all of the options that are available.

Reply:
add